.travis.yml

```diff
@@ -30,8 +30,8 @@ install:
   - git -C occa reset --hard $OCCA_HEAD
   - make -C occa -j2
   - export OCCA_DIR=$PWD/occa MFEM_DIR=$PWD/mfem PETSC_INSTALL=$HOME/install/petsc-3.9.3
-  - test -d $PETSC_INSTALL
-    || ( curl -O http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-3.9.3.tar.gz
+  - test -z | find $PETSC_INSTALL -name PETSc.pc
```

Comment:
Just specify the path `test -s "$PETSC_INSTALL/lib/pkgconfig/PETSc.pc"`.

Reply:
We could upgrade to PETSc 3.10 at any time. It shouldn't have any effect (just a slower Travis build the first time after upgrading).